src/test/java/com/training/services/tests/ProductsTest.java

```diff
@@ -0,0 +1,48 @@
+package com.training.services.tests;
+
+import com.training.base.BaseTest;
+import com.training.utilities.RestEngine;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class ProductsTest extends BaseTest {
+    Map<String, Object> headerMap;
+    Map<String, Object> queryParam;
+    String baseURL;
+    com.training.services.products.Response response;
+
+    @BeforeEach
+    public void setData() {
+        baseURL = map.get("url_base") + "/en/royal/mobile/v3/products";
+
+        headerMap = new HashMap();
+        headerMap.put(map.get("ContentTypeHeader"), map.get("ContentTypeValue"));
+        headerMap.put(map.get("AppKeyHeader"), map.get("AppKeyValue"));
+
+        queryParam = new HashMap();
+        queryParam.put(map.get("shipIdParam"), map.get("sailingIdParamValue"));
+        queryParam.put(map.get("categoryParam"), map.get("categoryParamValue"));
+    }
+
+    /**
+     * Shorex test, checking the response, checking total hits is 52 or not.
+     */
+    @Test
+    public void testShorexTotalHitsValidate() {
+        response = new RestEngine().getResponseGet(baseURL, headerMap, queryParam)
+                .as(com.training.services.products.Response.class);
```

Comment:
do we need a fully qualified name here? can't we just specify this in our imports?